api/build.gradle

```diff
@@ -30,7 +30,8 @@ sourceSets {
   }
 }
 dependencies {
-  compile group: 'uk.gov.hmcts.auth.totp', name: 'totp', version: '1.0.1'
+  compile project(':service-token-generator')
```

Comment:
is this needed?

Reply:
Absolutely not. Removed in 845b3ce.

Reply:
At this moment -- yes. Will be removed in the upcoming changes.